features/git-town/configuration/configuration.feature

```diff
@@ -9,28 +9,54 @@ Feature: listing the configuration
     Given I have configured the main branch name as "main"
     And my non-feature branches are "qa" and "staging"
     When I run `git town config`
-    Then I see "Main branch: main"
-    And I see "Non-feature branches: qa, staging"
+    Then I see
+      """
+      Main branch:
+      main
```

Comment:
I would leave the old output format for the main config output, I think that one was ideal. 

```
$ git town config
Main branch: main
Non feature branches: qa, staging
((branch names are highlighted through a color, for example cyan))

$ git town non-feature-branches   ((example with existing nfb))
qa
staging

$ git town non-feature-branches   ((example without existing nfb))
```

The first command is human facing. It prints the output in a way that is easy to read, and highlights the results so that they stand out.

The second and third commands are for scripts. No human operator will enter them given that there is the shorter `config` directive. They output data in a way that is easy to parse and manipulate.